DEVELOPMENT.md

```diff
@@ -11,16 +11,17 @@ To automate our release process as much as possible we're using
 the [conventional commits specification][ccs].
```

Comment:
You might like this tool https://github.com/smallhadroncollider/cmt :)

Reply:
Looks cool, but I'm usually wary of introducing a tool that dictates a whole commit workflow. As each of us has their preferred way of working, something like this could get in the way.

I'd rather be for something a bit more light-weight, like a husky plugin that maybe warns about commit styling, so that people can still use their preferred workflow/editor to make the commits.

Reply:
Oh, I thought the point of this being documented was that your team strictly follows it, which would be made easier if a tool was used that had a specific config.

I might have missed the point due to not seeing the whole context :sweat_smile:

Reply:
It's a bit more tricky than that: as per our current workflow we squash stuff into a single commit at the end of a PR. This squashed commit should always adhere to the spec, otherwise our automated changelog generation breaks.

To the automation it doesn't really matter how the individual commits were formatted in the branch. But we'd like to keep them tidy to make the reviewer's life easier.

Reply:
Will be looking into this: https://github.com/radicle-dev/radicle-upstream/issues/132